src/RocksKeyValue.cc

```diff
@@ -266,6 +266,26 @@ class DB : public KeyValue::DB {
         writes = std::make_unique<RocksKeyValue::WriteBatch>(db_);
         return Status::OK();
     }
+
+    Status get(KeyValue::CollectionHandle _coll,
+               const std::string& key,
+               std::string& value) const override {
+        auto coll = reinterpret_cast<rocksdb::ColumnFamilyHandle*>(_coll);
+        const rocksdb::ReadOptions r_options; // what should this be set to?
+        std::string v_tmp;
+        rocksdb::Status s = db_->Get(r_options, coll, key, &v_tmp);
+        value = v_tmp;
```

Comment:
Since this is a performance optimization, suggest `value = std::move(v_tmp);` or directly giving `&value` to `rocksdb::DB::Get`